docs/src/tutorials/qless.md

```diff
@@ -0,0 +1,145 @@
+```@example qless1
+using QRMumps, LinearAlgebra, SparseArrays, Printf
+
+# Initialize data
+m, n = 5, 7
+irn = [1, 3, 5, 2, 3, 5, 1, 4, 4, 5, 2, 1, 3]
+jcn = [1, 1, 1, 2, 3, 3, 4, 4, 5, 5, 6, 7, 7]
+val = [1.0, 2.0, 3.0, 1.0, 1.0, 2.0, 4.0, 1.0, 5.0, 1.0, 3.0, 6.0, 1.0]
+
+A = sparse(irn, jcn, val, m, n)
+b = [40.0, 10.0, 44.0, 98.0, 87.0]
+y_star = [16.0, 1.0, 10.0, 3.0, 19.0, 3.0, 2.0]
+y₁ = zeros(n)
+y₂ = zeros(m)
+y₃ = zeros(n)
+
+Δy₁ = zeros(n)
+Δy₂ = zeros(m)
+Δy₃ = zeros(n)
+r = zeros(m)
+
+# Initialize QRMumps
+qrm_init()
+
+# Initialize data structures
+spmat = qrm_spmat_init(A)
+spfct = qrm_spfct_init(spmat)
+
+# Specify not storing Q for the QR factorization
+qrm_set(spfct, "qrm_keeph", 0)
+
+# Perform symbolic analysis of Aᵀ and factorize Aᵀ = QR
+qrm_analyse!(spmat, spfct; transp='t')
+qrm_factorize!(spmat, spfct, transp='t')
+
+# Solve Rᵀy₁ =  b  
+qrm_solve!(spfct, b, y₁; transp='t')
+
+# Solve Ry₂ = y₁
+qrm_solve!(spfct, y₁, y₂; transp='n')
+
+# Overall, RᵀRy₂ = b. Equivalently, RᵀQᵀQRy₂ = b or AAᵀy₂ = b
+
+# Compute least norm solution of min ‖b - Ay‖
```

Comment:
I’m confused. This step doesn’t belong in the solution of the least-norm problem.

Reply:
See this comment. I am not sure if this answers your question ?

> Thanks @MaxenceGollier ! I believe there is a mistake in the first example: `y_star` and `y\_2` have to be of size n(=7) instead of m(=5). In fact you cannot compute `norm(A*y\_2-b)` because of incompatible dimensions. The actual least-norm solution is `y\_3=A'*y\_2`. Have I missed something?